pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -1787,6 +1787,69 @@ protected void internalSetSchemaAutoUpdateCompatibilityStrategy(SchemaAutoUpdate
             "schemaAutoUpdateCompatibilityStrategy");
     }
 
+    protected boolean internalGetSchemaValidationEnforced() {
```

Comment:
the logic here doesn't seem to correct to me. Also it seems to be copied from the set logic.

The correct logic should be just as simple as following:

```
protected boolean internalGetSchemaValidationEnforced(） {
    validateAdminAccessForTenant(namespaceName.getTenant());
    return getNamespacePolicies(namespaceName).schema_validation_enforced;
}
```